src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -315,7 +454,7 @@ class ElytraFlight : Module() {
 
         packetPitch = forwardPitch.value.toFloat()
         mc.player.capabilities.isFlying = true
-        mc.player.capabilities.flySpeed = speedCreative.value
+        mc.player.capabilities.flySpeed = getSpeed(speedCreative.value, false).toFloat()
```

Comment:
Why is false there? Can creative mode not boost?

Reply:
That is only for the control mode boost speed